src/main/java/org/hmhb/kml/DefaultKmlService.java

```diff
@@ -78,12 +78,6 @@ public DefaultKmlService(
         );
     }
 
-    private String temporaryConvert(String shape) {
-        // TODO - after John submits the counties SQL, remove the xml from the shape data
-        return shape.replace("<Polygon><outerBoundaryIs><LinearRing><coordinates>", "")
```

Comment:
Not shown in this PR (because too many changes in the file) is taking out the xml in the counties we already had, and adding the other 130ish counties (w/o xml in their data).